packages/remix/test/integration/test/server/loader.test.ts

```diff
@@ -9,23 +9,6 @@ import {
 jest.spyOn(console, 'error').mockImplementation();
 
 describe('Remix API Loaders', () => {
-  it('does not add a loader if there is not one defined.', async () => {
-    const baseURL = await runServer();
-    const url = `${baseURL}/`;
-    const envelope = await getEnvelopeRequest(url);
-    const transaction = envelope[2];
-
-    assertSentryTransaction(transaction, {
```

Comment:
Why did we delete this test?

Reply:
We no longer have a case where there's no `loader` in `root.tsx` defined.
Might need to utilize multiple projects to be able to test those kind of conflicted scenarios. We will probably need a similar thing to test manual wrappers too.

Trying to figure out now, will add that to this PR, if it doesn't require a large structural change.